src/robot/libdocpkg/model.py

```diff
@@ -65,18 +95,23 @@ def doc_format(self, format):
 
     @setter
     def inits(self, inits):
-        return self._add_parent(inits)
+        return self._sort_keywords(inits)
 
     @setter
     def keywords(self, kws):
-        return self._add_parent(kws)
+        return self._sort_keywords(kws)
 
-    def _add_parent(self, kws):
+    def _sort_keywords(self, kws):
```

Comment:
This helper does other things than sorting. Something more generic like `_process_keywords` would be a better name.